.run/SkijaInjectSample (show long frames) .run.xml

```diff
@@ -0,0 +1,25 @@
+<component name="ProjectRunConfigurationManager">
```

Comment:
Why it does belong to VCS?

Reply:
So everyone who uses IDEA can simply run it.

If they will get in the way, we can remove them, or group them into separate folder.

Reply:
Guess better way to provide predefine run config is Gradle task.

Reply:
After thinking about it more, I agree. Better to make them as tasks.

Will do this in another PR (there is another tasks that should be  rewriten).

I have working prototype.